setup.py

```diff
@@ -737,7 +737,7 @@ def is_package_excluded(package: str, exclusion_list: List[str]):
     'typing-extensions>=3.7.4;python_version<"3.8"',
     'tzlocal>=1.4,<2.0.0',
     'unicodecsv>=0.14.1',
-    'werkzeug<1.0.0',
+    'werkzeug~=1.1.0',
```

Comment:
```suggestion
    'werkzeug~=1.0, >=1.0.1',
```
I guess.

Also you should never trust me :D